lib/remap-functions/src/length.rs

```diff
@@ -0,0 +1,133 @@
+use remap::prelude::*;
+
+#[derive(Clone, Copy, Debug)]
+pub struct Length;
+
+impl Function for Length {
+    fn identifier(&self) -> &'static str {
+        "length"
+    }
+
+    fn parameters(&self) -> &'static [Parameter] {
+        &[Parameter {
+            keyword: "value",
+            accepts: |v| matches!(v, Value::Array(_) | Value::Map(_)),
```

Comment:
Does this need to be updated to include `Value::Bytes`?

Reply:
Good catch. Although, the runtime keyword type checking is a relic from the past that no longer does anything, since the checks happen at compile-time now.

We'll get rid of this eventually (and probably replace it with a more robust compile-time check in which functions define their argument types using the `Kind` type, instead of having to do it through the current `type_def` function impl (which is fraught with honest mistakes, as we've seen in the recent uptick in PRs, trying to fix those)